plugin/src/main/kotlin/com/jetbrains/packagesearch/plugin/ui/panels/tree/PackageSearchModulesTree.kt

```diff
@@ -58,7 +58,7 @@ fun PackageSearchModulesTree(
         onExpandAll = viewModel::expandAll,
         onCollapseAll = {
             val rootIds = tree.roots.map { it.id }.toSet()
-            viewModel.treeState.selectedKeys = viewModel.treeState.selectedKeys intersect rootIds
+            viewModel.treeState.selectedKeys =( viewModel.treeState.selectedKeys intersect rootIds).toList()
```

Comment:
Given that the keys are using lists, I assume that this is a quite old version...

Reply:
is in jewel foundation so nothing to do… we need to wait update to have the Set